portfolio/src/main/webapp/style.css

```diff
@@ -24,13 +24,15 @@
 
 .blog-section {
     align-items: center;
+    display: flex;
     flex-direction: column;
 }
 
 .blog-section-container {
     display: flex;
     flex-direction: row;
     margin-bottom: 100px;
+    height: 900px;
```

Comment:
We usually sort the css properties. But it isn't a hard requirement.

Reply:
got it!